### DISCUSSION TITLE
Missing strings from the django admin panel

### DISCUSSION THREAD

**Discussion by tomkolp** [WeblateOrg/weblate#5429] [Q&A] [answered]
I can't find the strings to translate anywhere in the weblate project.
![Screenshot_20210214-153628](https://user-images.githubusercontent.com/39633242/107879738-efcc4680-6eda-11eb-9cb9-8a3b14850032.png)

**Comment by nijel** [comment, marked as answer]
Some of that are coming from third-party modules (for example https://github.com/python-social-auth/), some are names of models which are not localizable unless extra code is added for that.

Given that our long-term plan is that Django admin interface will not be needed for using Weblate, no effort has been spent on localizing it. See https://github.com/WeblateOrg/weblate/issues/2363
